docs/deployment/metrics.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Metrics"
+layout: default
+sort: 7
+---
+
+# Metrics
+
+Metrics are configured to be exposed using prometheus.Operator API's by
+default. If you want to expose metrics using the prometheus.Operator
+API's you need to install the prometheus.Operator in your cluster.
+
+The exposed metrics are
+
+| Metric        | Type | Meaning |
+| --------------- | ---------------- | ---------------- |
+|  `nfd_master_build_info`           | Gauge | Version from which nfd-master was built. |
+|  `nfd_worker_build_info`           | Gauge | Version from which nfd-worker was built. |
+|  `nfd_updated_nodes`           |  Counter | Time taken to label a node |
+|  `nfd_crd_processing_time`          |  Gauge | Time taken to process a NodeFeatureRule CRD |
+| `nfd_feature_discovery_duration_seconds` |  HistogramVec | Time taken to discover features on a node |
+
+## Via Kustomize
+
+To allow [prometheus-operator][prometheus-operator]
+to scrape metrics from node-feature-discovery,
+run the following command:
+
+```bash
+kubectl apply -k https://github.com/kubernetes-sigs/node-feature-discovery/deployment/overlays/default?ref={{ site.release }}
+kubectl apply -k https://github.com/kubernetes-sigs/node-feature-discovery/deployment/overlays/prometheus?ref={{ site.release }}
+```
+
+## Via Helm
+
+We recommend setting
+`--set prometheus.prometheusSpec.podMonitorSelectorNilUsesHelmValues=false`
+when deploying prometheus-operator via Helm to enable the prometheus-operator
+to scrape metrics from any PodMonitor.
+
+## Chart parameters
```

Comment:
I think this section is redundant and not very relevant to the user. If we want to have this, I think one table (or list) for all parameters should be enough (these multiple sections take a disproportionately big mount of real estate on the page 😅)